sdk/formrecognizer/Azure.AI.FormRecognizer/src/TrainingOperationExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.FormRecognizer.Training
+{
+    /// <summary>
+    /// </summary>
+    public static class TrainingOperationExtensions
```

Comment:
Should we merge all extension methods into one class?

Reply:
These are visible from `.Training` without including the root namespace, so yes, will consolidate to a single class.